app/Controller/PagesController.php

```diff
@@ -72,5 +118,59 @@ public function display() {
 			}
 			throw new NotFoundException();
 		}
-	}
+  }
+
+  private $顧客名;
+
+  function set顧客名($v)
+  {
+    $this->顧客名 = $v;
+    return $this;
+  }
+
+  function get顧客名()
+  {
+    return $this->顧客名;
+  }
+}
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
```

Comment:
__[phpmd]__
* The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.